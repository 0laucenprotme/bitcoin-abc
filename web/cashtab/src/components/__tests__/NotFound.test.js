import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../NotFound';

test('Render NotFound component', () => {
    const component = renderer.create(<NotFound />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
