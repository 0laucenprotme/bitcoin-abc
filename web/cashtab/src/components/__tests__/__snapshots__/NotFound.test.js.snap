// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotFound component 1`] = `
<div
  className="ant-row-flex ant-row-flex-center"
  style={Object {}}
>
  <div
    className="ant-col ant-col-8"
    style={Object {}}
  >
    <h1>
      Page not found
    </h1>
  </div>
</div>
`;
